FDRS_Gateway/FDRS_Gateway.ino

```diff
@@ -132,12 +130,20 @@ void loop() {
     getSerial();
   }
   getLoRa();
-#ifdef USE_WIFI
+  #ifdef USE_WIFI
   if (!client.connected()) {
-    DBG("Connecting MQTT...");
-    reconnect();
+    reconnect(1, true);
   }
-  client.loop();
+  client.loop(); // for recieving incoming messages and maintaining connection
+
+  timeClient.update();  //update internal clock if possible
```

Comment:
Is this updating every loop? Should we maybe just update the time when it connects?